fission/src/ui/modals/configuring/SettingsModal.tsx

```diff
@@ -76,91 +66,88 @@ const SettingsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
-            />
-            <Slider
-                min={1}
-                max={15}
-                value={PreferencesSystem.getGlobalPreference<number>("PitchSensitivity")}
-                label={"Pitch Sensitivity"}
-                format={{ maximumFractionDigits: 2 }}
-                onChange={(_, value) => setPitchSensitivity(value as number)}
-            />
-            <Slider
-                min={1}
-                max={15}
-                value={PreferencesSystem.getGlobalPreference<number>("YawSensitivity")}
-                label={"Yaw Sensitivity"}
-                format={{ maximumFractionDigits: 2 }}
-                onChange={(_, value) => setYawSensitivity(value as number)}
-            />
-            <Label size={LabelSize.Medium}>Preferences</Label>
-            <Checkbox
-                label="Report Analytics"
-                defaultState={PreferencesSystem.getGlobalPreference<boolean>("ReportAnalytics")}
-                onClick={checked => {
-                    setReportAnalytics(checked)
-                }}
-            />
-            <Checkbox
-                label="Use Metric"
-                defaultState={PreferencesSystem.getGlobalPreference<boolean>("UseMetric")}
-                onClick={checked => {
-                    setUseMetric(checked)
-                }}
-            />
-            <Checkbox
-                label="Render Score Zones"
-                defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderScoringZones")}
-                onClick={checked => {
-                    setRenderScoringZones(checked)
-                }}
-            />
-            <Checkbox
-                label="Render Scene Tags"
-                defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderSceneTags")}
-                onClick={checked => {
-                    setRenderSceneTags(checked)
-                    if (!checked) new SceneOverlayEvent(SceneOverlayEventKey.DISABLE)
-                    else new SceneOverlayEvent(SceneOverlayEventKey.ENABLE)
-                }}
-            />
-            <Checkbox
-                label="Render Scoreboard"
-                defaultState={PreferencesSystem.getGlobalPreference<boolean>("RenderScoreboard")}
-                onClick={checked => {
-                    setRenderScoreboard(checked)
-                }}
-            />
+            <div className="flex overflow-y-auto flex-col gap-2 bg-background-secondary rounded-md p-2 min-w-[22vw]">
+                <Label size={LabelSize.Medium}>Screen Settings</Label>
+                <Dropdown
+                    label="Quality Settings"
+                    options={qualitySettingsOptions}
+                    defaultValue={PreferencesSystem.getGlobalPreference<string>("QualitySettings")}
+                    onSelect={selected => {
+                        setQualitySettings(selected)
+                    }}
+                />
+                <Box minHeight={"5px"} />
```

Comment:
This might not be helpful but maybe we could add a Component that is called Spacer that just adds space
it might help for code readability